src/app.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
```

Comment:
In this component you should import **configureStore** function from **'./invision/store';** and after that create a new store for **Invision** component using configureStore function: 
```
import configureStore from './invision/store';

const App = (
    <Invision Router={BrowserRouter} store={configureStore(window.PRELOADED_STATE)} />
  )
```